RecoMTD/DetLayers/plugins/BTLDetLayerGeometryBuilder.cc

```diff
@@ -33,26 +35,33 @@ vector<DetLayer*> BTLDetLayerGeometryBuilder::buildLayers(const MTDGeometry& geo
         const GeomDet* geomDet = geo.idToDet(BTLDetId(side, tray, module, 0, 1));
         if (geomDet != nullptr) {
           geomDets.push_back(geomDet);
-          LogTrace(metname) << "get BTL module " << std::hex << BTLDetId(side, tray, module, 0, 1).rawId() << std::dec
-                            << " at R=" << geomDet->position().perp() << ", phi=" << geomDet->position().phi();
+#ifdef EDM_ML_DEBUG
+          edm::LogVerbatim(metname) << "get BTL module " << std::hex << BTLDetId(side, tray, module, 0, 1).rawId()
```

Comment:
LogVerbatim with EDM_ML_DEBUG has the same behavior as LogTrace.
Is there a substantial problem in enabling the debug/trace level via MessageLogger? 
Info is not on by default either; so, I'd guess that ability to enable is there.

Reply:
it just requires less manipulations of the default MessageLogger configuration to be selected in my view.